Source/Scene/GltfLoader.js

```diff
@@ -883,6 +858,33 @@ function loadAttribute(
       attribute.byteOffset = 0;
       attribute.byteStride = undefined;
       attribute.quantization = vertexBufferLoader.quantization;
+
+      if (loadBuffer) {
+        attribute.buffer = vertexBufferLoader.buffer;
+      }
+
+      if (loadTypedArray) {
+        attribute.typedArray = ComponentDatatype.createArrayBufferView(
+          vertexBufferLoader.quantization.componentDatatype,
+          vertexBufferLoader.typedArray.buffer
+        );
+      }
```

Comment:
Could we factor out a function to contain this logic? this section of code is getting rather nested.